src/game/process/cmd/teams.js

```diff
@@ -0,0 +1,63 @@
+import { TEAMS } from '../../common/const.mjs';
+
+export function assignToTeam(state, walletAddress) {
+  const teamsConfig = state.teamsConfig;
+  const teams = teamsConfig.teams;
+  const amount = teamsConfig.amount;
+  if (teams.length < amount) {
```

Comment:
hmm, wouldn't it be better to initilize all the teams in the initState function (instead of doing it "lazily" in this function)?